Inception/ClientApp/src/App/Services.ts

```diff
@@ -0,0 +1,123 @@
+import { LinkTestResult, SiteTestResult } from "./Models";
+import { Observable } from "rxjs/Observable";
+import { Subject } from "rxjs/Subject";
+import { Injectable } from "@angular/core";
+
+@Injectable()
+export class Service {
```

Comment:
Each service should be in separate file (and desirable have descriptive name)

Reply:
+